Sources/MacroToolkit/NormalizedType.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+public enum NormalizedType: TypeProtocol, SyntaxExpressibleByStringInterpolation {
+    /// A composition of two types (e.g. `Encodable & Decodable`). Used to
+    /// combine protocol requirements.
+    case composition(NormalizedCompositionType)
+    /// A some or any protocol type (e.g. `any T` or `some T`).
+    case someOrAny(NormalizedSomeOrAnyType)
+    /// A function type (e.g. `() -> ()`).
+    case function(NormalizedFunctionType)
+    /// An implicitly unwrapped optional type (e.g. `Int!`).
+    case implicitlyUnwrappedOptional(NormalizedImplicitlyUnwrappedOptionalType)
+    /// A member type (e.g. `Array<Int>.Element`).
+    case member(NormalizedMemberType)
+    /// A placeholder for invalid types that the resilient parser ignored.
+    case missing(NormalizedMissingType)
+    /// A pack expansion type (e.g. `repeat each V`).
+    case packExpansion(NormalizedPackExpansionType)
+    /// A pack reference type (e.g. `each V`).
+    case packReference(NormalizedPackReferenceType)
+    /// A simple type (e.g. `Int` or `Box<Int>`).
+    case simple(NormalizedSimpleType)
+    /// A suppressed type in a conformance position (e.g. `~Copyable`).
+    case suppressed(NormalizedSuppressedType)
+    //// A tuple type (e.g. `(Int, String)`).
+    case tuple(NormalizedTupleType)
+    
+    public var _baseSyntax: TypeSyntax {
+        let type: any TypeProtocol = switch self {
+        case .composition(let type as any TypeProtocol),
+                .someOrAny(let type as any TypeProtocol),
+                .function(let type as any TypeProtocol),
+                .implicitlyUnwrappedOptional(let type as any TypeProtocol),
+                .member(let type as any TypeProtocol),
+                .missing(let type as any TypeProtocol),
+                .packExpansion(let type as any TypeProtocol),
+                .packReference(let type as any TypeProtocol),
+                .simple(let type as any TypeProtocol),
+                .suppressed(let type as any TypeProtocol),
+                .tuple(let type as any TypeProtocol):
+            type
```

Comment:
Indent this case block by one more level (just a stylistic choice I've made for the rest of the repo)

Reply:
Not the standard indentation, but fixed in 71aff30.